sdks/python/apache_beam/transforms/enrichment_handlers/bigquery.py

```diff
@@ -194,14 +202,16 @@ def __call__(self, request: Union[beam.Row, List[beam.Row]], *args, **kwargs):
               "Make sure the values passed in `fields` are the "
               "keys in the input `beam.Row`." + str(e))
         values.extend(current_values)
-        requests_map.update((val, req) for val in current_values)
+        requests_map.update(
+            (self.create_row_key(req), req) for val in current_values)
```

Comment:
Is this now just inserting `(self.create_row_key(req), req)` into the map repeatedly? It doesn't seem like we need the `for val in current_values` anymore